### PR TITLE
Added Python Primer Button 

### LINKED ISSUE #415: User study - There is no link to the Python primer doc
No link to python primer doc (for client-version only)

### PR DESCRIPTION
**Pull request by theresa-lee**
Fixes #415

### CODE REVIEW

File: client/question/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -105,9 +105,12 @@ tie.directive('learnerView', [function() {
                     </ui-codemirror>
                   </div>
                 </div>
-                <select class="tie-select-menu" name="lang-select-menu" ng-class="{'night-mode': isInDarkMode}">
+                <select ng-if="SERVER_URL" class="tie-select-menu" name="lang-select-menu" ng-class="{'night-mode': isInDarkMode}">
                   <option value="Python" selected>Python</option>
                 </select>
+                <button ng-if="!SERVER_URL" class="tie-python-primer tie-button" ng-class="{'night-mode': isInDarkMode}">
+                  <a class="tie-primer-link" ng-class="{'night-mode': isInDarkMode}" target="_blank" ng-href="../docs/py-primer-{{isInDarkMode === true ? 'dark' : 'light'}}.html">New to python?</a>
```
Comment (seanlip) [inline client/question/components/LearnerViewDirective.js]: Could you extract this into $scope.getPythonPrimerUrl()?

Also, note that you don't need the `=== true` bit. `isInDarkMode ? 'dark' : 'light'` should suffice.

**Comment by rabidbit** [comment]
I think a link (in the same location and font size) might be better than a button. It will likely see less use that the other buttons; so, I'm thinking a link (using standard web link color) would look softer / less glaring. Also, a link points to a doc / resource as opposed to performing some action on the application or UI.

**Comment by theresa-lee** [comment]
Yeah, I was thinking how the language selector would actually be a button dropdown too so I was trying to keep it consistent with that. The functionality is all here though, I don't mind it being a link. I wanted to submit this before I left so whether you want to change the CSS now on this PR or in another PR after this one is submitted is up to you?

File: client/question/components/LearnerViewDirective.js [hunk 2]
```diff
@@ -916,6 +927,14 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+         * Provides the URL to the appropriately themed python primer file.
+         */
+        $scope.getPythonPrimerUrl = function() {
+          var primerTheme = $scope.isInDarkMode ? 'dark' : 'light';
+          return "../docs/py-primer-" + primerTheme + ".html";
```
Comment (seanlip) [inline client/question/components/LearnerViewDirective.js]: Nit: use single quotes please.

Comment (theresa-lee) [reply client/question/components/LearnerViewDirective.js]: Done!

**Comment by seanlip** [comment]
Thanks @theresa-lee! @rabidbit please feel free to merge if you're happy with this.

**Comment by rabidbit** [comment]
We can do the CSS after this PR.

**Pull request merged**